bin/satpy_launcher.py

```diff
@@ -79,9 +79,10 @@ def process(msg, prod_list):
     for wrk in config['workers']:
         cwrk = wrk.copy()
         try:
-            cwrk.pop('fun')(job, **cwrk)
+            func = wrk('fun')
+            func(job, **cwrk)
```

Comment:
how does this work if `fun` isn't removed from `cwrk` ?

Reply:
I hoped it would? I'll add the `pop` in there.

Reply:
`pop` added